statics/index.html

```diff
@@ -3,10 +3,11 @@
   <head>
     <meta charset="utf-8"/>
     <title>Mughsot Visual Report</title>
+    <link rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
If you want to use bootstrap then [install](https://www.npmjs.com/package/bootstrap) it locally.